tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/DataType/ContentPicker.spec.ts

```diff
@@ -1,81 +1,70 @@
-﻿import {test} from '@umbraco/playwright-testhelpers';
+﻿import {ConstantHelper, NotificationConstantHelper, test} from '@umbraco/playwright-testhelpers';
 import {expect} from "@playwright/test";
 
 const dataTypeName = 'Content Picker';
-let dataTypeDefaultData = null;
-let dataTypeData = null;
+const customDataTypeName = 'Custom Content Picker';
+const editorAlias = 'Umbraco.ContentPicker';
+const editorUiAlias = 'Umb.PropertyEditorUi.DocumentPicker';
 
 test.beforeEach(async ({umbracoUi, umbracoApi}) => {
   await umbracoUi.goToBackOffice();
   await umbracoUi.dataType.goToSettingsTreeItem('Data Types');
-  dataTypeDefaultData = await umbracoApi.dataType.getByName(dataTypeName);
+  await umbracoApi.dataType.ensureNameNotExists(customDataTypeName);
 });
 
 test.afterEach(async ({umbracoApi}) => {
-  if (dataTypeDefaultData !== null) {
-    await umbracoApi.dataType.update(dataTypeDefaultData.id, dataTypeDefaultData);
-  }
+  await umbracoApi.dataType.ensureNameNotExists(customDataTypeName);
 });
 
-test('can show open button', async ({umbracoApi, umbracoUi}) => {
+// Remove fixme when the front-end is ready. Currently the "Show open button" setting is removed
```

Comment:
This was removed by Front End on purpose, if you merge the newest changes from 15dev then this test should be removed